workflow/rules/post-analysis.smk

```diff
@@ -4,30 +4,36 @@ rule preseq_lc_extrap:
     output:
         "results/preseq/{sample}.lc_extrap"
     params:
-        "-v -seed 1"
+         "-v {} -seed 1".format( "" if config["single_end"] else "-pe" )
     log:
         "logs/preseq/{sample}.log"
     wrapper:
         "0.64.0/bio/preseq/lc_extrap"
 
 rule collect_multiple_metrics:
     input:
-         bam="results/orphan_rm_sorted/{sample}.bam",
+         bam="results/filtered/{sample}.sorted.bam",
          ref="resources/ref/genome.fasta"
     output: #ToDo: add descriptions to report captions
         # Through the output file extensions the different tools for the metrics can be selected
         # so that it is not necessary to specify them under params with the "PROGRAM" option.
         # Usable extensions (and which tools they implicitly call) are listed here:
         #         https://snakemake-wrappers.readthedocs.io/en/stable/wrappers/picard/collectmultiplemetrics.html.
+        [
+           "{path}{sample}.insert_size_metrics",
+            report(
+                "{path}{sample}.insert_size_histogram.pdf",
+                caption="../report/plot_insert_size_histogram_picard_mm.rst",
+                category="MulitpleMetrics"
```

Comment:
There's a typo here (and in every other place this category is used, it seems;). And while we fix the typo, I think we could also go for the more readable version with a space included:
```suggestion
                category="Multiple Metrics (picard)"
```